sass/index.scss

```diff
@@ -59,1298 +143,506 @@ $white: #FFFFFF;
+            font-size: 1.5rem;
             font-weight: bold;
-            text-decoration: none;
         }
-    }
-}
 
+        // email field and submit button
+        #pre-reg-form { 
+            display: flex;
+            position: relative;
+            flex-wrap: wrap;
+            margin-top: 1rem; // from mini-window.p
+            justify-content: center;
 
-// Nav
-nav {
-    background-color: white;
-    border-bottom: 2px solid rgba($light-blue, 25%);
-    display: flex;
-    justify-content: space-between;
-    font-size: var(--body-font-size);
-    height: 100px;
-    position: fixed;
-    padding: 0px 85px 0px 8rem;
-    top: 0;
-    left: 0;
-    right: 0;
-    z-index: 1;
-
-    .top, .left, .right {
-        display: flex;
-        align-items: center;
-    }
-
-    .left {
-        flex-grow: 1;
-    }
+            // elements on same row w/o line break
+            white-space: nowrap;
 
-    #toggle {
-        margin-right: 2rem;
-        color: $blue;
-        font-size: 2rem;
-        width: 2rem;
-        display: none;
-    }
+            // takes up rest of mini window
+            flex-grow: 1;
+        }
+        input[name="email"], input[type="submit"] {
+            display: inline-block;
+            max-height: 43px;
+            height: 100%;
+            justify-self: center;
+            flex: 1 1 auto;
+            
+            margin: 0 5px; // 5 pixels apart left/right
+            font-family: 'Work Sans', sans-serif;
+            position: relative;
+        }
 
-    a {
-        margin-right: 2rem;
-        text-decoration: none;
-        color: $blue;
-    }
+        input[name="email"] {
+            max-width: 438px;
+            width: 438px;
 
-    .link {
-        border-bottom: 2px solid white;
-        transition: all 0.2s;
+            font-size: 1.3rem;
+            padding-left: 10px;
+            border: 3px white solid;
+            margin-bottom: auto; // fills the rest of the bottom
 
-        &:not(:focus):hover {
-            // underline upon hover, disable hover upon click
-            font-weight: bold;
-            border-color: $blue;
+            &:focus {
+                outline: 3px $desat-blue solid;
+            }
         }
 
-        &.active {
-            font-weight: bold;
-            color: $darker-blue;
-            border-color: $darker-blue;
+        input[type="submit"] {
+            max-width: 106px;
+            width: 106px;
+
+            font-size: 1.3rem;
+            border: 3px solid white;
+            opacity: 70%;
 
             &:hover {
-                border-color: $darker-blue;
+                color: $off-white;
+                background-color: $desat-blue;
             }
         }
     }
 
-    #login {
-        @include button-primary;
-    }
+    .title-bar {
 
-    #logo {
-        width: 80px;
-        border: none;
+        // Main container
+        display: flex;
+        width: 100%;
+        max-height: 42px;
```

Comment:
For future: again, I'd not set the height so we can wrap down on smaller screens without having to increase it